gcloud-java-dns/src/main/java/com/google/gcloud/dns/ProjectInfo.java

```diff
@@ -0,0 +1,285 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.dns;
+
+import static com.google.api.client.repackaged.com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.common.base.MoreObjects;
+
+import java.io.Serializable;
+import java.math.BigInteger;
+import java.util.Objects;
+
+/**
+ * The class that encapsulates information about a project in Google Cloud DNS. A project is a top
+ * level container for resources including {@code ManagedZone}s. Projects can be created only in the
+ * APIs console.
+ *
+ * @see <a href="https://cloud.google.com/dns/api/v1/projects">Google Cloud DNS documentation</a>
+ */
+public class ProjectInfo implements Serializable {
+
+  private static final long serialVersionUID = 8696578863323485036L;
+  private final String id;
+  private final BigInteger number;
+  private final Quota quota;
+
+  /**
+   * This class represents quotas assigned to the {@code ProjectInfo}.
+   *
+   * @see <a href="https://cloud.google.com/dns/api/v1/projects">Google Cloud DNS documentation</a>
```

Comment:
Would be good if we can provide pointers on how to change the quota for DNS related settings.